src/Draggable/Emitter/Emitter.js

```diff
@@ -66,7 +66,14 @@ export default class Emitter {
     }
 
     if (caughtErrors.length) {
-      throw new Error(`Draggable caught errors while triggering '${event.type}'`, caughtErrors);
+      /* eslint-disable no-console */
```

Comment:
How do we feel about this? Should we elevate it to a global rule and update `eslintrc.js`?

Reply:
So far we are only using it here, let's wait and see if you'll start using it more, what do you think?

Reply:
yep that sounds fine